mock-rollout/src/Products/ProductContext.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { createContext, FC, useEffect, useState } from 'react';
```

Comment:
Let's refactor the folders structure.

create a components folder and move ProductItem & List to that folder
create a store folder and move the Context there
create a model folder and move the model there
create a services folder and move the service there - pay attention that the purpose for a service is to use a **generic** mechanism (i.e http-service, file-service, firebase-service, notification-service etc), and this services are used to handle a specific entity in the project). To be more specific, ProductsService, it's not a real service, but it uses one, so the get function implemented in this should be moved to the context/ provider